port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -204,6 +258,12 @@ namespace port_drayage_plugin
             ROS_DEBUG_STREAM("cargo id: " << *_latest_mobility_operation_msg.cargo_id);
         }
         else{
+            // If this message is for 'PICKUP', then the 'cargo_id' field is required
+            if(_latest_mobility_operation_msg.operation == PORT_DRAYAGE_PICKUP_OPERATION_ID) {
+                ROS_DEBUG_STREAM("Received 'PICKUP' operation, but no cargo_id was included.");
+                throw std::invalid_argument("Received 'PICKUP' operation, but no cargo_id was included.");
```

Comment:
Same as above. To be honest, might be more reasonable to simply ROS_LOG these at WARN or something. Anomalous conditions for sure, but does it prevent vehicle operation entirely?